### PR TITLE
Bugfix

### PR DESCRIPTION
**Pull request by Nyholm**
... at least I thought.

### CODE REVIEW

**Comment by Nyholm** [comment]
Tag a new patch release after merge.

File: src/ArrayCachePool.php [hunk 1]
```diff
@@ -27,9 +27,9 @@ class ArrayCachePool extends AbstractCachePool
     /**
      * @param array $cache
      */
-    public function __construct(array $cache = [])
+    public function __construct(array &$cache = [])
     {
-        $this->cache = $cache;
+        $this->cache = &$cache;
```
Comment (cryptiklemur) [inline src/ArrayCachePool.php:32]: Why the by reference, objects are always passed by reference

Comment (cryptiklemur) [inline src/ArrayCachePool.php:32]: does it not pass arrays?

Comment (Nyholm) [inline src/ArrayCachePool.php:32]: Arrays are not objects. Arrays are passed by value. I want to be able to do like this: https://github.com/php-cache/array-adapter/blob/master/tests/IntegrationTagTest.php#L23

That is why I need the reference.

**Pull request merged**